data/controls.js

```diff
@@ -4,34 +4,34 @@
  * http://mozilla.org/MPL/2.0/.
  */
 
-document.querySelector('.controls').style.display = 'none';
-document.querySelector('.video-wrapper').style.display = 'none';
+document.querySelector(".controls").style.display = "none";
```

Comment:
yeah let's do single quotes to be consistent with testpilot, otherwise I'll get caught up on it all the time :)